org.eclipse.xtext.tests/src-gen/org/eclipse/xtext/parsetree/reconstr/serializer/SerializationErrorTestLanguageSemanticSequencer.java

```diff
@@ -59,7 +59,7 @@ public void sequence(ISerializationContext context, EObject semanticObject) {
 	 *     Indent returns Indent
 	 *
 	 * Constraint:
-	 *     ((((req=TwoRequired opt=TwoOptions) | opt=TwoOptions)? indent+=Indent+) | indent+=Indent+)?
```

Comment:
Hmm ...

Reply:
dont blame the messenger ;)
do you think that is a regression in the serializer generator?

Reply:
> dont blame the messenger ;)

;) I don't think it's a regression.